AuxTel/Troubleshooting/General-Troubleshooting/AuxTel-Rotator-Out-Of-Range.rst

```diff
@@ -0,0 +1,105 @@
+
+.. _AuxTel-Rotator-Out-Of-Range-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+During the imaging or spectroscopic surveys, the scripts latiss_acquire (Spectroscopic survey) or `track_target_and_take_image` (Imaging survey) fail with the following error:
+
+.. code-block:: text
+
+    Error in run Traceback (most recent call last): 
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-6.0.0/lib/python3.10/site-packages/lsst/ts/salobj/base_script.py", line 603, in do_run await self._run_task 
+    File "/net/obs-env/auto_base_packages/ts_standardscripts/python/lsst/ts/standardscripts/base_track_target_and_take_image.py", line 237, in run await self.track_target_and_setup_instrument() 
+    File "/net/obs-env/auto_base_packages/ts_standardscripts/python/lsst/ts/standardscripts/auxtel/track_target_and_take_image.py", line 141,  in track_target_and_setup_instrument await self.atcs.slew_icrs( 
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/base_tcs.py", line 647, in slew_icrs raise ack_error 
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/base_tcs.py", line 615, in slew_icrs await self.slew( 
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/base_tcs.py", line 814, in slew raise ack_err 
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/base_tcs.py", line 803, in slew await self._slew_to( 
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/atcs.py", line 1413, in _slew_to await slew_cmd.start(timeout=slew_timeout) 
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-6.0.0/lib/python3.10/site-packages/lsst/ts/salobj/topics/remote_command.py", line 487, in start return await cmd_info.next_ackcmd(timeout=timeout) 
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-6.0.0/lib/python3.10/site-packages/lsst/ts/salobj/topics/remote_command.py", line 191, in next_ackcmd raise base.AckError(msg="Command failed", ackcmd=ackcmd) 
+    lsst.ts.salobj.base.AckError: msg='Command failed', ackcmd=(ackcmd private_seqNum=1630455441, ack=<SalRetCode.CMD_FAILED: -302>, error=6611,  
+    result='Rejected : Rotator out of range. Target in rotator limit (-170 to 170 degrees) but out of slew limit margin (1 degs)') 
+
+
+
+.. _AuxTel-Rotator-Out-Of-Range-Procedure-Steps:
+
+Procedure Steps
+===============
+
+The interim solution requires recovering the ATPtg back to ``ENABLE`` in the case it went to ``FAULT``, and skiping to the next target in the ATQueue by pressing the :guilabel:`PLAY` buttom. 
+If this script also fails with the same error, wait three mintues and try again.
+
+
+1. Recover ATPtg from ``FAULT`` state: Transition the ATPtg CSC from LOVE ASummaryState through the states, ``FAULT`` → ``STANDBY`` → ``START`` → ``ENABLE``
```

Comment:
For numbered bullets could you use # as in for instance AuxTel/Troubleshooting/General-Troubleshooting/AuxTel-Elevation-out-of-range.rst?

Reply:
I tried to do it like this the first time but it seems that when there is something in between (such as a code-block) the numbering resets to 1. That's why I decided to use numbers instead. Don't know if there is a way to solve this issue.